app/services/save_practice_service.rb

```diff
@@ -133,4 +135,19 @@ def crop_main_display_image
       @practice.main_display_image.reprocess!
     end
   end
+
+  def update_initiating_facility
+    @initiating_facility_type = @practice_params[:initiating_facility_type]
```

Comment:
nit: we don't need the `@` here for `@initiating_facility_type` and `@initiating_facility`